spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/autoconfig/TraceBraveAutoConfiguration.java

```diff
@@ -59,13 +59,12 @@
  * @since 3.0.0
  */
 @Configuration(proxyBeanMethods = false)
-@ConditionalOnProperty(value = "spring.sleuth.enabled", matchIfMissing = true)
-@EnableConfigurationProperties(SleuthProperties.class)
-@Import({ TraceBaggageConfiguration.class, SamplerAutoConfiguration.class })
+@OnBraveEnabled
+@EnableConfigurationProperties({ SleuthProperties.class, SleuthSpanFilterProperties.class,
+		SleuthBaggageProperties.class, SleuthTracerProperties.class, SleuthBaggageProperties.class })
+@Import({ BraveBridgeConfiguration.class, BraveBaggageConfiguration.class, SamplerConfiguration.class,
+		BraveHttpConfiguration.class })
 @AutoConfigureBefore(TraceAutoConfiguration.class)
-@AutoConfigureAfter(TraceBravePropagationAutoConfiguration.class)
-// public allows @AutoConfigureAfter(TraceAutoConfiguration)
-// for components needing Tracing
 public class TraceBraveAutoConfiguration {
```

Comment:
Curious why this can't be called `BraveAutoConfiguration`. Isn't Brave unique enough (with its package) to refer to tracing?